MAUI/ImageEditor/getting-started.md

```diff
@@ -224,6 +224,8 @@ public partial class MainPage : ContentPage
 
 ![GettingStarted in .NET Maui ImageEditor](images\getting-started\imageeditor-gettingstarted.png)
 
+N> If we set the Stream source with a local variable, the stream will be closed after it is used by the image, and we cannot process the stream again. So we recommend using stream images using the Lamba function so that we can process them whenever needed.

```

Comment:
If we set the Stream source with a local variable, the stream will be closed after it is used by the image, and we cannot process the stream again. So we recommend using stream images by creating a new stream instance inside Lamba function so that we can process them whenever needed.

imageEditor.Source = ImageSource.FromStream(() => new MemoryStream(imageBytes))

Reply:
Updated.